example_test.go

```diff
@@ -2,17 +2,16 @@ package nut
 
 import (
 	"fmt"
-
-	nut "github.com/robbiet480/go.nut"
```

Comment:
Can you revert these changes? The example_test.go file needs them as is.

Reply:
Done